src/richie/apps/courses/templates/courses/cms/course_fragment.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n cms_tags %}
```

Comment:
The already existing fragment template files are using another naming parttern. Let's follow this naming pattern and name the current file something like: fragment_course_glimpse.html?